fbw-a32nx/src/systems/instruments/src/PFD/AltitudeIndicator.tsx

```diff
@@ -310,20 +341,41 @@ export class AltitudeIndicatorOfftape extends DisplayComponent<AltitudeIndicator
                 <g ref={this.normal} style="display: none">
                     <path id="AltTapeOutline" class="NormalStroke White" d="m117.75 123.56h17.83m-4.7345-85.473v85.473m-13.096-85.473h17.83" />
                     <MinimumDescentAltitudeIndicator bus={this.props.bus} />
-                    <SelectedAltIndicator bus={this.props.bus} />
+                    <SelectedAltIndicator bus={this.props.bus} selectedAltitude={this.shownTargetAltitude} altitudeColor={this.targetAltitudeColor} />
                     <AltimeterIndicator bus={this.props.bus} altitude={this.altitude} />
-                    <MetricAltIndicator bus={this.props.bus} />
+                    <MetricAltIndicator bus={this.props.bus} targetAlt={this.shownTargetAltitude} altitudeColor={this.targetAltitudeColor} />
                     <path id="AltReadoutBackground" class="BlackFill" d="m130.85 85.308h-13.13v-8.9706h13.13v-2.671h8.8647v14.313h-8.8647z" />
                     <RadioAltIndicator bus={this.props.bus} filteredRadioAltitude={this.props.filteredRadioAltitude} />
                     <DigitalAltitudeReadout bus={this.props.bus} />
                 </g>
             </>
         );
     }
+
+    private handleAltManagedChange() {
+        // TODO find proper logic for this (what happens when a constraint is sent by the fms but vertical mode is not managed)
+        const isSelectedModeActive = this.activeVerticalMode !== VerticalMode.OP_CLB && this.activeVerticalMode !== VerticalMode.OP_DES
```

Comment:
Small comment: this looks like it's true if **not** in a selected mode, i.e. if we are in a managed mode, which makes sense considering the usage